roles/matrix-bridge-mautrix-whatsapp/templates/config.yaml.j2

```diff
@@ -105,19 +95,16 @@ bridge:
     default_bridge_receipts: true
     default_bridge_presence: true
     # Servers to always allow double puppeting from
-    double_puppet_server_map: "{{ matrix_domain }}"
-
+    double_puppet_server_map:
```

Comment:
Shouldn't we push this to a `matrix_mautrix_whatsapp_bridge_double_puppet_server_map` variable?